### PR TITLE
chore: 'show me how' button in feature showcase toaster

### PR DESCRIPTION
**Pull request by Harshita-mindfire**
This PR aims to add a `Show me how` button in the Feature showcase toaster. Currently, we have this for showcasing Graph Themes. 

# Dendron Extended PR Checklist

### Basics

- [x] code should follow [Code Conventions](dev.process.code)
- [x] circular dependency check: make sure your code is not introducing new circular dependencies in plugin-core.  See [Avoiding Circular Dependencies](dev.process.code.best-practices).
- [x] sticking to existing conventions instead of creating new ones
  - eg: [if configuration for utilities are already in one module or package, add future utilities there as well](https://github.com/dendronhq/dendron/pull/1960#discussion_r786228021)

### Extended
- General
  - [~] check whether code be simplified
  - [~] check if similar function already exist in the codebase. if so, can it be re-used?
  - [~] check if this change adversely impact performance
- Operations
  - [~] when shipping this change, will it just work or will it introduce additional operational overhead due to complicated interface or known bugs?
- Architecture
  - [~] check if code is introducing changes on a foundational class or interface. if so, call for design review if needed
    - eg: [making changes to DNode](https://github.com/dendronhq/dendron/pull/2158#pullrequestreview-854689586)

### Basics
- [~] if you are adding analytics related changes, make sure the [Telemetry](https://wiki.dendron.so/notes/84df871b-9442-42fd-b4c3-0024e35b5f3c.html) docs are updated

### Extended
- [~] can we track the performance of this change to know if it is _successful_?
  - eg: [see usage for export pod](https://github.com/dendronhq/dendron/pull/2190#pullrequestreview-855715612)

### Basics

- [~] [Write Tests](dev.process.qa.test) 
- [ ] [Confirm existing tests pass](dev.process.qa.test)
- [x] [Confirm manual testing](dev.process.qa.test) 
- [x] Common cases tested
- [ ] 1-2 Edge cases tested
- [~] If your tests changes an existing snapshot, snapshots have been [updated](dev.process.qa.test)

### Extended
NA

## Docs
NA

### Extended
NA

### CODE REVIEW

File: packages/plugin-core/src/showcase/GraphThemeTip.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class GraphThemeTip implements IFeatureShowcaseMessage {
   getDisplayMessage(displayLocation: DisplayLocation): string {
```
Comment (kevinslin) [inline packages/plugin-core/src/showcase/GraphThemeTip.ts:23]: since we are doing this more often, can we create a command that is enabled in development called "Show Tip" which chooses what tip is currently shown? will help with debugging. can be a next item

File: packages/plugin-core/src/views/ShowMeHowView.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import vscode from "vscode";
+
+const getShowMeHowViewHtml = ({ name, src }: { name: string; src: string }) => {
+  return `<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (kevinslin) [inline packages/plugin-core/src/views/ShowMeHowView.ts:8]: can this expand to take the full width of the web view?

https://www.loom.com/i/b508dd004d054f68b6754301bf8e87e1

**Review by kevinslin — CHANGES_REQUESTED** [review]
nice job, some minor comments. as an aside, can you also post the gif in #showcase channel in discord? also send it to @jpknwls so he can tweet it :)

File: packages/plugin-core/src/showcase/GraphThemeTip.ts [hunk 3]
```diff
@@ -23,7 +23,7 @@ export class GraphThemeTip implements IFeatureShowcaseMessage {
...
     switch (displayLocation) {
       case DisplayLocation.InformationMessage:
-        return `New themes for Graph View! Change the appearance of the note graph by clicking the config button on the top left corner of the graph view and selecting one of the built-in styles. You can even customize the appearance to your liking with css.`;
```
Comment (jonathanyeung) [inline packages/plugin-core/src/showcase/GraphThemeTip.ts:26]: I think it'll be good to A/B this one.  Can you put the deleted text / button as an A group and the .gif flow in a B group?

Comment (Harshita-mindfire) [reply packages/plugin-core/src/showcase/GraphThemeTip.ts:26]: added

File: packages/plugin-core/src/views/ShowMeHowView.ts (hunk 2 above)
Comment (Harshita-mindfire) [reply packages/plugin-core/src/views/ShowMeHowView.ts:8]: I did try this before, but the gif gets really blurry when expanded to the full width. An alternative is to add a higher resolution gif, but it is around 3MB.
Meanwhile, I did center align the gif with the web view and embedded it inside an anchor which links to the loom video.

**Review by kevinslin — APPROVED** [review]

**Review by SeriousBug — APPROVED** [review]

**Review by jonathanyeung — APPROVED** [review]
